addons/beehave/nodes/leaves/blackboard_compare.gd

```diff
@@ -0,0 +1,39 @@
+@tool
+class_name BlackboardCompareCondition extends ConditionLeaf
+
+
+enum Operators {
+	EQUAL,
+	NOT_EQUAL,
+	GREATER,
+	LESS,
+	GREATER_EQUAL,
+	LESS_EQUAL,
+}
+
+
+@export var left_operand: String = ""
+@export_enum("==", "!=", ">", "<", ">=", "<=") var operator: int = 0
+@export var right_operand: String = ""
+
+
+@onready var _left_expression: Expression = parse_expression(left_operand)
```

Comment:
Interesting, that `@export var expression: Expression` does not work. I'd love some sort of custom UI in Godot that would allow us to do that.